actors/providers/ldap/pom.xml

```diff
@@ -65,11 +65,13 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <artifactId>orbit-actors-json</artifactId>
             <version>${project.version}</version>
         </dependency>
+
         <dependency>
-            <groupId>org.apache.directory.api</groupId>
-            <artifactId>api-all</artifactId>
-            <version>1.0.0-M30</version>
+            <groupId>org.apache.directory.server</groupId>
+            <artifactId>apacheds-all</artifactId>
```

Comment:
Actually this is what I tried first @johnou :), but there is an issue with the apacheds and api-ldap-client-api where apacheds finds "multiple schemas" at the classpath and fails to run. I tried to use maven exclusion to solve the issue, but did not worked. After several tries I just gave up, but you can give a try. :)